src/System.Private.CoreLib/shared/System/String.Manipulation.cs

```diff
@@ -34,7 +34,7 @@ private static unsafe void FillStringChecked(string dest, int destPos, string sr
             }
         }
 
-        public static string Concat(object? arg0)
+        public static string? Concat(object? arg0)
```

Comment:
This just came up in a conversation with @gafter earlier today.  I'd be tempted to say we should fix this implementation to never return null.  Every other Concat overload checks for ToString returning null, substituting Empty in that case... seems like we should do the same here, e.g.
```C#
return arg0?.ToString() ?? string.Empty;
```
I don't actually know who calls this method, other than compiler-generated code.

@jkotas?

Reply:
Good idea.

Reply:
Sounds fine to me.